solutions/7.js

```diff
@@ -0,0 +1,11 @@
+const getDivisors = (number, i=1, max=0) =>{
+  if(i === number){
+    return max;
+  }
+  if(number%i === 0 && i>max){
```

Comment:
You don't need `&& i>max` here because `i`  is always greater than or equal to `max`